### DISCUSSION TITLE
article/z7btimbk/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/pengzhanbo.cn#56] [Blog-Comment]
# article/z7btimbk/

@property CSS at-rule是 CSS Houdini API 的一部分，它允许开发者显式地定义他们的 CSS 自定义属性, 允许进行属性类型检查、设定默认值以及定义该自定义属性是否可以被继承。 @property 的出现，极大的增强了 CSS 的能力。 CSS Houdini CSS Houdini 开放 CSS 的底层 API 给开发...

https://pengzhanbo.cn/article/z7btimbk/

**Comment by mxj5** [comment]
测试评论
